tests/integration_test/admin_controller.py

```diff
@@ -450,6 +453,71 @@ def execute_actions(self, site_launcher, actions):
                         client_ids = list(site_launcher.client_properties.keys())
                     for cid in client_ids:
                         site_launcher.start_client(cid)
+
+            elif command == "test":
+                if args[0] == "admin_commands":
+                    print(("\n" + "*" * 120) * 20)
+                    print("\n" + "=" * 40)
+                    print("\nRunning through tests of admin commands:")
+                    print("\n" + "=" * 40)
+                    print("\nCommand: set_timeout")
+                    print(self.admin_api.set_timeout(11).get("details").get("message"))
+                    print("\nActive SP:")
+                    print(self.admin_api.get_active_sp().get("details"))
+                    print("\nList SP:")
+                    print(self.admin_api.list_sp().get("details"))
+                    print("\nCommand: get_available_apps_to_upload")
+                    print(self.admin_api.get_available_apps_to_upload())
+                    print("\nList Jobs:")
+                    list_jobs_return_message = self.admin_api.list_jobs().get("details").get("message")
+                    print(list_jobs_return_message)
+                    first_job = list_jobs_return_message.split()[14]
+                    print("\nCommand: ls server -a .")
```

Comment:
can these prints be changed to assert so it will fail automatically if something is not expected?